netbox/project-static/styles/custom/_markdown.scss

```diff
@@ -29,7 +29,7 @@
 }
 
 // Remove the bottom margin of <p> elements inside a table cell
-td > .rendered-markdown {
+td > .rendered-markdown, .form-text .rendered-markdown {
```

Comment:
We should break out the `p:last-of-type` rule as it applies generally, whereas the `max-height` and `overflow-y` rules apply only to Markdown content within a table cell.